XInfoGroups() return error "Conn has unread data" when entries-read is nil

Results running on my machine:
```shell
127.0.0.1:6379> xinfo groups mq
1)  1) "name"
    2) "cg1"
    3) "consumers"
    4) (integer) 0
    5) "pending"
    6) (integer) 0
    7) "last-delivered-id"
    8) "0-0"
    9) "entries-read"
   10) (nil)
   11) "lag"
   12) (integer) 6
2)  1) "name"
    2) "g1"
    3) "consumers"
    4) (integer) 0
    5) "pending"
    6) (integer) 0
    7) "last-delivered-id"
    8) "0-0"
    9) "entries-read"
   10) (nil)
   11) "lag"
   12) (integer) 6
```
Among them, the "entries-read" field is nil

When I write code with the corresponding command:
```go
import (
	"context"
	"github.com/go-redis/redis/v9"
	"testing"
)

func Test(t *testing.T) {
	c := redis.NewClient(&redis.Options{
		Addr: "localhost:6379",
	})
	cmd := c.XInfoGroups(context.Background(), "mq")
	res, err := cmd.Result()
	if err != nil {
		t.Fatal(err)
	}
	t.Logf("%+v\n", res)
}
```
the result:
```shell
=== RUN   Test
redis: 2022/06/23 15:18:14 pool.go:356: Conn has unread data
    test_test.go:25: redis: nil
--- FAIL: Test (0.01s)
```

--- a/command.go
+++ b/command.go
@@ -2047,7 +2047,7 @@ func (cmd *XInfoGroupsCmd) readReply(rd *proto.Reader) error {
 				}
 			case "entries-read":
 				group.EntriesRead, err = rd.ReadInt()
-				if err != nil {
+				if err != nil && err != Nil {
 					return err
 				}
 			case "lag":

